pkgs/applications/graphics/dungeondraft/default.nix

```
@@ -0,0 +1,77 @@
+{ stdenv
+, lib
+, requireFile
+, dpkg
+, xorg
+, libGL
+, udev
+, pulseaudio
+, libkrb5
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dungeondraft";
+  version = "1.1.0.3";
+
+  src = requireFile {
+    name = "Dungeondraft-${version}-Linux64.deb";
+    url = "https://dungeondraft.net/";
+    sha256 = "08am7zi47qqfd9sdxp57a4qh6ln9kkdk60xgyjl1xdz80nc63w5f";
```

Use SRI hash instead, and use `hash` instead of `sha256` as it is more idiomatic:
```suggestion
    hash = "sha256-rvBhmAXotx6o9K8DM9ucyVIDMVGn3N50ag7jQ+I/VSE=";
```

This was obtained with:
`nix hash to-sri --type "sha256"  08am7zi47qqfd9sdxp57a4qh6ln9kkdk60xgyjl1xdz80nc63w5f`